analysis/wwz/make_datacards.py

```diff
@@ -377,13 +383,16 @@ def main():
         sample_names_dict_mc["UL16"]    = sg.create_mc_sample_dict("UL16")
         sample_names_dict_mc["UL17"]    = sg.create_mc_sample_dict("UL17")
         sample_names_dict_mc["UL18"]    = sg.create_mc_sample_dict("UL18")
+    if run == "run3":
+        sample_names_dict_mc["2022"] = sg.create_mc_sample_dict("2022")
```

Comment:
Can you align the equals sign here (and below) with the ones a couple lines up? Sorry...